tests/resources/functions/node-logs-fail/index.js

```diff
@@ -0,0 +1,16 @@
+module.exports = async (context) => {
```

Comment:
Do we need this? Cant we reuse existing `node-logs`?

Reply:
In order to reuse the existing method, we would need to pass an additional param to tell it to throw an exception. Let me know if you like that approach better.